zuul.d/ansible-cloud-jobs.yaml

```diff
@@ -503,18 +448,3 @@
         override-checkout: stable-2.14
     vars:
       ansible_collections_repo: github.com/ansible-collections/pravic
-
```

Comment:
If I am not mistaken, this job https://github.com/ansible/ansible-zuul-jobs/pull/1892/files#diff-1651516334d52994025e65b62ea1ce5b2db20a48aa48e89b55c682049450e66eR411 can also be removed.